src/mito2/src/error.rs

```diff
@@ -309,8 +309,28 @@ pub enum Error {
         location: Location,
     },
 
+    #[snafu(display("Failed to get metadata from file {}, reason: {}", file, reason))]
+    NoKeyValue {
```

Comment:
the name `NoKeyValue` seems not match the description, do you have a better name?

Reply:
I'll change the description. It indicates a key-value pair could not be found in parquet's key value meta.